### PR TITLE
Manage environments page

### PR DESCRIPTION
**Pull request by SueSmith**
First draft of new environments page - pics to be dropped in later. Haven't touched the main variables page yet but will tackle that when I have a handle on what this page is going to include.

### CODE REVIEW

**Comment by SueSmith** [comment]
@kinlane starting to rework the variables doc to reflect environment rbac and the new managing environments page, reframing the stuff around sessions to "sharing and persisting".. feedback appreciated! 🙂

**Comment by SueSmith** [comment]
@claireriley I feel like the stuff around only being able to do xyz with edit access to an environment might be really repetitive, trying to make each section understandable if it's read in isolation.. This is a big change in general so I no longer have any idea if it's sensibly structured or readable.. 🤪 If you could give it a good check over that would be amazing 🙏

File: src/pages/docs/postman/variables-and-environments/managing-environments.md [hunk 1]
```diff
@@ -266,6 +266,9 @@ If you have edit access to an environment, you can update the variable values fr
 
 When you edit the initial value of a shared environment variable, your updated value will be reflected for everyone who has access to the environment, so ensure that you only do this when you are happy for your value to be synced with the Postman servers.
 
+> If you uncheck (deselect) a variable in your environment, it will only be available to collaborators who also have edit access to the environment (and its enabled / disabled status will be reflected for them). Anyone with viewer role on the environment will not see the unchecked variable.
```
Comment (claireriley) [inline src/pages/docs/postman/variables-and-environments/managing-environments.md:269]: "Anyone with viewer role" -> "Anyone with a/the viewer role" or "viewer permissions" maybe?

File: src/pages/docs/postman/variables-and-environments/variables.md [hunk 2]
```diff
@@ -198,6 +198,8 @@ pm.environment.set("variable_key", "variable_value");
 
 > If you have view but not edit access to an environment, your script code will only affect the current value, and will not be synced or shared with your team.
 
+You can use `unset` to remove a variable.
```
Comment (claireriley) [inline src/pages/docs/postman/variables-and-environments/variables.md:201]: Wondering if this should be a note > or maybe could include a brief snippet example?

**Pull request merged**